lib/sms_autofill.dart

```diff
@@ -38,26 +38,26 @@ class SmsAutoFill {
     await _channel.invokeMethod('unregisterListener');
   }
 
-  Future<String> get getAppSignature async {
-    final String appSignature = await _channel.invokeMethod('getAppSignature');
+  Future<String?> get getAppSignature async {
```

Comment:
I'll check native side the reason we can have null here, might be better to return an error instead of null (the only case I can think of having null here is error cases)